src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+  src: url();
+}
+
+* {
+  margin: 0;
```

Comment:
don't use such rules as margin on `*` its have a bad influence on the performance of your page